Source/JavaScriptCore/yarr/YarrParser.h

```diff
@@ -1168,9 +1168,9 @@ class Parser {
         return ch;
     }
 
-    inline UChar32 consumeAndCheckIfValidClassSetCharacter()
+    inline int consumeAndCheckIfValidClassSetCharacter()
     {
-        UChar32 ch = consumePossibleSurrogatePair<UnicodeParseContext::PatternCodePoint>();
+        int ch = consumePossibleSurrogatePair<UnicodeParseContext::PatternCodePoint>();
```

Comment:
Why is this int?  I'd expect char32_t or uint32_t

Reply:
This function returns either a character or `-1`. Throughout this file, the type `int` is used for this. It’s traditional, in fact. The function `fgetc` from classic `stdio.h` is one of the many places that have used this in the past. This just happens to be one call site in this file where someone wrote `UChar32` instead of `int`, an inconsequential mistake because the two are names for the same type.

We could rework this to use a different strategy, but I held back on that for now.

Reply:
I changed this to `char32_t`.